README.md

```diff
@@ -7,141 +7,99 @@ Java NETCONF Client.
 2. Add the location of jnc.py to the `$PYANG_PLUGINPATH` environment variable, or
 3. Use the `--plugindir` option of pyang each time you want to use JNC
 
-If more than one of these approaches is used, you will end up with optparse
-conflicts so please choose one and stick with it. From here on, we will assume
-that you went for (1), but using (2) or (3) should be analogous.
+**Note!**
+If more than one of these approaches is used, you will end up with *optparse* conflicts, so please choose one and stick with it. From here on, we will assume that you went for (1), but using (2) or (3) should be analogous.
 
 JNC plugin can be used to generate Java classes from a YANG file of your choice.
-There are a collection of yang files in the 'examples/yang' directory. To
-generate classes for a YANG file, open a terminal, change directory to where
-you want the classes to be generated, launch pyang with the jnc format,
-specifying the output folder and the yang model file.
-
-For example, to generate the classes for simple.yang to the 'examples'
-directory with base package gen.simple, type:
 
-     $ pyang -f jnc --jnc-output src/gen/simple yang/simple.yang
+There is a collection of yang files in the 'examples/yang' directory.
 
-There should now be a newly generated 'src' folder in the current directory,
-containing a directory structure with the generated classes. Note that 'src' is
-special treated so that it does not become part of the package names of the
-generated classes.
+To generate classes for a YANG file, open a terminal, change directory to where you want the classes to be generated, launch pyang with the `"jnc"` format, specifying the output folder and the yang model file.
 
-To get more detailed information on how the generation proceeds the --jnc-debug
-or --jnc-verbose options can be used. Rerunning JNC silently overwrites any old
-classes in the output directory.
-            
-### JNC Java application
+For example, to generate the classes for `simple.yang` file included in our `examples` directory with base package name `gen.simple`, type:
 
-To actually use the generated classes, you need to compile Java client code
-using the JNC library (JAR file). The examples of JNC application can be 
-found in the `examples` directory.
-
-Gradle is used to build application. It is also shown how to use JNC
-pyang plugin with Gradle task. 
+     $ pyang -f jnc --jnc-output src/gen/simple yang/simple.yang
 
-See `README` for the examples. 
+There should now be a newly generated `"src"` folder in the current directory, containing a directory structure with the generated classes. Note that `"src"` is special treated so that it does not become part of the package names of the generated classes.
 
+To get more detailed information on how the generation proceeds the `--jnc-debug` or `--jnc-verbose` options can be used. Re-running JNC silently overwrites any old classes in the output directory.
```

Comment:
I think this section should mention that there's that gradle plugin (available for anyone who uses gradle).

Reply:
updated in latest commit, in Java application build section